Owncloud iOs Client/Login/Login/WebLoginViewController.swift

```diff
@@ -55,6 +57,9 @@ import Foundation
         // Dispose of any resources that can be recreated.
     }
     
+    func cancelBarButtonPressed(){
```

Comment:
Use the previous cancelButtonTapped, we need to keep the unwind to allow retrieve auth data. Otherwise we will need to set delegates and removes segues.